apis/Google.LongRunning/Google.LongRunning/OperationsClientPartial.cs

```diff
@@ -34,6 +35,19 @@ public virtual IScheduler Scheduler
         {
             get { throw new NotImplementedException(); }
         }
+
+        // It would be nice to make this internal, but then anyone creating their
+        // own subclass for testing would be broken.
+
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="callSettings"></param>
+        /// <returns></returns>
```

Comment:
Docs will be there when I write tests :) Basically we need to get a GAX release done before I can finish this PR.